census/models/utils.js

```diff
@@ -141,7 +141,11 @@ var processEntries = function(data, options) {
 
     data.pending = _.where(data.entries, {'isCurrent': false, 'reviewed': false});
     data.rejected = _.where(data.entries, {'isCurrent': false, 'reviewed': true, 'reviewResult': false});
-    _.remove(data.entries, function(e) { return e.isCurrent === false; });
```

Comment:
@kravets-levko this and below lines need explicit unit tests in https://github.com/okfn/opendatacensus/blob/develop/tests/models.js